t/900-configparsing.t

```diff
@@ -15,6 +15,7 @@ my $naemon = "$buildroot/src/naemon/naemon";
 my $etc = "$buildroot/t/etc";
 my $precache = "$buildroot/t/var/objects.precache";
 
+plan skip_all => 'config check won\' run as root user' if $> == 0;
```

Comment:
Rather than skipping the test we could instead pass the --allow-root argument to the Naemon binary.